clients/drcachesim/tracer/raw2trace.h

```diff
@@ -112,13 +199,7 @@ class raw2trace_t {
                        void *process_cb_user_data, void (*free_cb)(void *data));
 
     /**
-     * Performs the first step of do_conversion() without further action: parses and
-     * iterates over the list of modules.  This is provided to give the user a method
-     * for iterating modules in the presence of the custom field used by drmemtrace
-     * that prevents direct use of drmodtrack_offline_read().
-     * On success, calls the \p process_cb function passed to handle_custom_data()
-     * for every module in the list, and returns an empty string at the end.
-     * Returns a non-empty error message on failure.
+     * Constructs a #module_mapper_t.
```

Comment:
A little elaboration seems in order: "... which parses and iterates the list of modules using the callbacks passed to handle_custom_data().  Returns a non-empty..."